### PR TITLE
Batch edits

### PR DESCRIPTION
**Pull request by Reettaphant**
Allow for prepending/appending to image descriptions when batch editing images which have different descriptions.
screen shot 2019-02-22 at 14 28 14

### CODE REVIEW

**Comment by paperboyo** [comment]
💎 

I would, maybe, spread them radios to fill the whole width to prevent accidental clicks?

File: kahuna/public/js/components/gr-info-panel/gr-info-panel.html [hunk 1]
```diff
@@ -143,11 +136,41 @@
...
+                      >
+                        <div
+                         editable-radiolist="ctrl.descriptionOption"
+                         e-ng-options="option.value as option.text for option in ctrl.descriptionOptions"
+                         e-name="descriptionOption"
+                    >
+                        </div>
+                    </div>
+                    <div ng-if="ctrl.userCanEdit && descriptionEditForm.$visible" class="editable-buttons">
+                        <button class="button-cancel" type="button" ng-click="descriptionEditForm.$cancel()">
```
Comment (akash1810) [inline kahuna/public/js/components/gr-info-panel/gr-info-panel.html]: can we add a comment around why we've re-implemented the buttons?

File: kahuna/public/js/edits/service.js [hunk 2]
```diff
@@ -286,8 +287,32 @@ service.factory('editsService',
...
+           (
+             descriptionOption !== descriptionEditOptions.append.value &&
+             descriptionOption !== descriptionEditOptions.prepend.value
+            )
+        ) {
+          return value;
+        }
+
+        if (descriptionOption === descriptionEditOptions.append.value) {
+          return imageAccessor.readMetadata(image).description + ' ' + value;
```
Comment (akash1810) [inline kahuna/public/js/edits/service.js]: should we trim value here?

**Comment by paperboyo** [comment]
Couple of small things:
- Shift+Enter doesn’t allow for entering new line after prepended and before appended text (subsequent soft returns do work)
- Usual committing edits by Cmd+Enter doesn’t work

**Comment by franziskas** [comment]
@paperboyo please raise a new health task for those

**Pull request merged**